girder_worker/app.py

```diff
@@ -182,7 +182,7 @@ def __call__(self, *args, **kwargs):
 
 
 def _maybe_model_repr(obj):
-    if hasattr(obj, '__repr_model_') and hasattr(obj._repr_model_, '__call__'):
+    if hasattr(obj, '_repr_model_') and hasattr(obj._repr_model_, '__call__'):
```

Comment:
`and callable(obj._repr_model_)`

Reply:
I wanted to avoid callable because it is deprecated (i.e. not available) in python 3.0 and 3.1. hasattr is technically [wrong](https://stackoverflow.com/a/18704793),  but it is consistently wrong across python versions

Reply:
will use https://pythonhosted.org/six/#six.callable